apps/hoarder/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+services:
+  web:
```

Comment:
This id should match the `id` in the `config.json` file, so it should be `hoarder`